src/game.cpp

```diff
@@ -850,6 +850,15 @@ void Game_Prepare()
         engine_camera.transform.M4x4[12 + 2] += player->character->height;
         engine_camera.transform.angles[0] = player->transform.angles[0] + 180.0f;
         engine_camera.current_room = player->self->room;
+
+        // Equip pistols if they are available in inventory
+        if(Inventory_GetItemsCount(player->inventory, ITEM_PISTOL) > 0)
```

Comment:
todo: (for myself, do not forget) check player->character->set_weapon_model_func != NULL